### PR TITLE
Dom querying - Bryan Brown

### PR DESCRIPTION
**Pull request by bbrown949**
Screenshot 2023-04-03 at 9 48 15 AM

### CODE REVIEW

File: dom-querying/main.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+console.log('hello, world');
+const test = document.querySelector('h1');
```
Comment (shawnkost) [inline dom-querying/main.js]: `test` isn't a very good variable name, it should be more representative of what this variable actually holds.

File: dom-querying/main.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+console.log(test);
+console.dir(test);
+const test2 = document.querySelector('#explanation');
```
Comment (shawnkost) [inline dom-querying/main.js]: `test2` isn't a very good variable name, it should be more representative of what this variable actually holds.

File: dom-querying/main.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+console.log(test2);
+console.dir(test2);
+const test3 = document.querySelector('.hint');
```
Comment (shawnkost) [inline dom-querying/main.js]: `test3` isn't a very good variable name, it should be more representative of what this variable actually holds.

File: dom-querying/main.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+console.log(test3);
+console.dir(test3);
+const selectorAll = document.querySelectorAll('example-link');
+console.log(selectorAll);
```
Comment (shawnkost) [inline dom-querying/main.js]: You're missing one more `document.querySelectorAll`. Check out the instructions again

**Review by shawnkost — CHANGES_REQUESTED** [review]
A few changes requested here Bryan!

File: dom-querying/main.js (hunk 1 above)
Comment (bbrown949) [reply dom-querying/main.js]: updated

**Review by shawnkost — CHANGES_REQUESTED** [review]
Still missing one more `document.querySelectorAll`. Make sure you check out the instructions again

File: dom-querying/main.js (hunk 4 above)
Comment (bbrown949) [reply dom-querying/main.js]: 🤦🏻‍♂️ added

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**